sdk/include/cardboard.h

```diff
@@ -126,6 +140,18 @@ void Cardboard_initializeAndroid(JavaVM* vm, jobject context);
 
 /// @}
 
+/////////////////////////////////////////////////////////////////////////////
+// Screen Parameters
+/////////////////////////////////////////////////////////////////////////////
+/// @defgroup screen-parameters Screen Parameters
+/// @brief This module calculates the screen size and current screen orientation
+/// @{
+/// Returns the current device screen orientation
+///
+/// @return         CardboardScreenOrientation enum
+CardboardScreenOrientation CardboardScreenParameters_getScreenOrientation();
```

Comment:
We should add a `@pre`requisite explaining that if the SDK is not initialized, it will always return `kUnknown`.